changelog/1504.trivial.rst

```diff
@@ -0,0 +1,3 @@
+Set the maximum version of ``click`` to be less than ``8.1`` in the
+pre-commit_ configuration for compatibility with black_ version
+``21.12b0``.
```

Comment:
Since `click` isn't a package dependency and we're only changing the CI here (no changes to the `plasmapy` pkg), I'd argue we we don't need (and probably shouldn't have) a change log for this.

Reply:
I was on the fence about that, so I'm okay with deleting the changelog entry (which I just did).